### PR TITLE
Create a dropdown menu that uses the kButton

### PR DESCRIPTION
**Pull request by christianmemije**
### Summary

Addressing #2412 

screenshot at 2018-02-23 19-21-27

screenshot at 2018-02-23 19-20-51

…

### Reviewer guidance

Test them. Find flaws please.

…

### References

Addressing #2412 

----

### Contributor Checklist

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] Contributor has fully tested the PR manually
- [ ] Screenshots of any front-end changes are in the PR description
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/core/assets/src/views/buttons-and-links/k-button.vue [hunk 1]
```diff
@@ -61,6 +67,14 @@
...
         default: 'button',
       },
+      /**
+       * Adds a dropdown arrow
+       */
+      hasDropdown: {
+        type: Boolean,
+        required: false,
+        default: false,
+      },
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/buttons-and-links/k-button.vue:78]: let's make this a private prop

File: kolibri/core/assets/src/views/k-dropdown-menu/index.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    },
+    props: {
+      text: {
+        type: String,
+        required: true,
+      },
+      appearance: {
+        type: String,
+        required: false,
+        default: 'raised-button',
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/k-dropdown-menu/index.vue]: you can add a validator here, re-using this one: https://github.com/learningequality/kolibri/pull/3257/files#diff-6f198d4d8621b85cdf03813e2db84ee6R24

File: kolibri/core/assets/src/views/buttons-and-links/k-button.vue (hunk 1 above)
Comment (christianmemije) [reply kolibri/core/assets/src/views/buttons-and-links/k-button.vue:78]: Made private!

**Comment by christianmemije** [comment]
localhost_8000_style_guide 1
localhost_8000_style_guide 2

**Pull request merged**